docs/issue_template.md

```diff
@@ -0,0 +1,22 @@
+# Checklist for author
+
+Remove items that aren't applicable.
+
+## Issue Type
+
+- [ ] Bug Report
+- [ ] Feature Request (requested by stakeholders)
+- [ ] Feature Proposal (possible improvements)
```

Comment:
I'm not sure I know what the difference between request and proposal is :)

Could we just add a list of the feature type labels we use here and instruct the user to pick one? Bug, Feature, Enhancement, Investigate?

Reply:
This was just my mind trying to think of some good naming on a Friday evening, I was thinking of more like "what the client has requested" vs "what we propose to add" but your naming conventions are way better

Reply:
I guess request is a "must have", and proposal is a "would be nice to have"? I think labelling would be our task based on our investigation but I also don't feel like there's a need to distinguish between request and proposal. Given that rems users (i.e. non-technical persons) would create issues still requires training on how to create an issue anyway.